en/docs/reference/mediators/log-mediator.md

```diff
@@ -38,29 +39,14 @@ follows.
 </ul></td>
 </tr>
 <tr class="even">
+<td><strong>Message</strong></td>
 <td>
-<p><strong>Level</strong></p>
-</td>
-<td>
-<p>This parameter is used to specify the log level. The possible values are as follows.</p>
-<ul>
-<li><strong>Custom</strong> : If this is selected, only the properties added to the Log mediator configuration will be logged.</li>
-<li><strong>Full</strong> : If this is selected, all the standard headers logged at the <strong>Simple</strong> level as well as the full payload of the message will be logged. This log level causes the message content to be parsed and hence incurs a performance overhead.</li>
-<li><strong>Simple</strong> : If this is selected, the standard headers (such as <code>To</code>, <code>From</code>, <code>WSAction</code>, <code>SOAPAction</code>, <code>ReplyTo</code>, and <code>MessageID</code>) will be logged.</li>
-<li><strong>Headers</strong> : If this is selected, all the SOAP header blocks will be logged.</li>
-</ul>
-<p>The properties included in the Log mediator configuration will be logged regardless of the log level selected.</p>
-</td>
-</tr>
-<tr class="odd">
-<td><strong>Separator</strong></td>
-<td>
-<p>This parameter specifies the value used to separate attributes in the log. By default, the separator is a comma (<code>,</code>).</p></td>
+<p>This parameter is used to define a log message. You can embed <a href="{{base_path}}/reference/synapse-properties/synapse-expressions">Synapse Expressions</a> in the template to generate the log message.</p></td>
 </tr>
 </tbody>
 </table>
 
-You can add properties inside the Log mediator to log additional information. The parameters available to configure a property are as follows:
+You can use the <code>Additional Parameters</code> section to log additional information. The configurations available under <code>Additional Parameters</code> section of the Log mediator are as follows.
```

Comment:
Let's use bold formatting for UI text instead of `<code>` tags.

Reply:
Fixed with 115eb27